### PR TITLE
FIX HTMLEditorField::setRows with Elemental

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-elemental/issues/1132

### CODE REVIEW

File: client/src/legacy/HtmlEditorField.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ ss.editorWrappers.tinyMCE = (function() {
     getConfig: function() {
       var selector = "#" + editorID,
         config = jQuery(selector).data('config'),
+        height = config.row_height ? config.row_height : '400px',
```
Comment (ChloeSartorelli) [inline client/src/legacy/HtmlEditorField.js]: Where is that `400px` value coming from? Seems fairly arbitrary? I assume this is replacing what the deleted line below used to do?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: client/src/legacy/HtmlEditorField.js (hunk 1 above)
Comment (sabina-talipova) [reply client/src/legacy/HtmlEditorField.js]: `400px` is the default value that we calculate on the server side if the setRows method was not applied. Yes, you're right, I'll change the logic here.

Comment (sabina-talipova) [reply client/src/legacy/HtmlEditorField.js]: Updated

File: client/src/legacy/HtmlEditorField.js [hunk 2]
```diff
@@ -83,6 +86,7 @@ ss.editorWrappers.tinyMCE = (function() {
     getConfig: function() {
       var selector = "#" + editorID,
         config = jQuery(selector).data('config'),
+        height = config.row_height ? config.row_height : 'undefined',
```
Comment (sabina-talipova) [inline client/src/legacy/HtmlEditorField.js:89]: Set to `undefined` if `row_height` is not set, `null` or `0`.

File: client/src/legacy/HtmlEditorField.js [hunk 3]
```diff
@@ -66,6 +66,9 @@ ss.editorWrappers.tinyMCE = (function() {
     },
 
     setHeight: function (event, height) {
+      if (typeof height === 'undefined') {
+        return;
```
Comment (sabina-talipova) [inline client/src/legacy/HtmlEditorField.js:70]: If `height` is `undefined` we should apply default TinyMCE configuration.

File: client/src/legacy/HtmlEditorField.js (hunk 2 above)
Comment (ChloeSartorelli) [inline client/src/legacy/HtmlEditorField.js:89]:
```suggestion
        height = config.row_height ? config.row_height : undefined,
```
The `undefined` type must not have quotes, or else it'll be `typeof string`

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: client/src/legacy/HtmlEditorField.js (hunk 2 above)
Comment (sabina-talipova) [reply client/src/legacy/HtmlEditorField.js:89]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM, works as expected.

**Pull request merged**